### PR TITLE
CLDR-16094 Change v42 spec to approved status

### PR DESCRIPTION
**Pull request by AEApple**
CLDR-16094

* Remove draft next to version number
* Make sure all sections have the approve status line
* Update version that the keyboard spec update is expected to v43
* Update all tables to be markdown tables

- [x] This PR completes the ticket.

### CODE REVIEW

**Comment by AEApple** [comment]
So the one thing I wasn't sure about was whether the PersonNames section should stay as draft spec in this release?

**Review by rickmcgowan — APPROVED** [review]
The changes all seem good to me.

**Comment by rickmcgowan** [comment]
No. The /document itself /is not a draft. It's being /released/ as part 
of the whole spec for Version 42.

For discussion: I'd suggest for this go-round for this major new feature 
is adding some explanatory paragraph at the top of the PersonNames part 
of the spec to indicate the drafty status of the *data* (the first 
release of spec and data, or whatever the situation is) for this 
particular release.

E.g., perhaps something along the lines of: "This document is new for 
CLDR release 42, and this is the first release of the specification and 
corresponding data. The documentation herein may not be as mature as 
other sections of the specification at this time. [Please report bugs 
yada yada boilerplate stuff here.]

R.

**Comment by pedberg-icu** [comment]
>So the one thing I wasn't sure about was whether the PersonNames section should stay as draft spec in this release?

@Annemarie No, as Rick noted, the doc status itself is not draft

> For discussion: I'd suggest for this go-round for this major new feature
is adding some explanatory paragraph at the top of the PersonNames part
of the spec to indicate the drafty status of the *data* (the first
release of spec and data, or whatever the situation is) for this
particular release.

@rickmcgowan We already have that, see the block quote in https://cldr-smoke.unicode.org/spec/maint-42/ldml/tr35-personNames.html#Introduction

File: docs/ldml/tr35-collation.md [hunk 1]
```diff
@@ -2,11 +2,9 @@
 
 # Unicode Locale Data Markup Language (LDML)<br/>Part 5: Collation
 
-<!-- HTML: no header -->
-<table><tbody>
- <tr><td>Version</td><td><b>42 (draft)</b></td></tr>
-<tr><td>Editors</td><td>Markus Scherer (<a href="mailto:markus.icu@gmail.com">markus.icu@gmail.com</a>) and <a href="tr35.md#Acknowledgments">other CLDR committee members</a></td></tr>
-</tbody></table>
+|Version|42              |
```
Comment (pedberg-icu) [inline docs/ldml/tr35-collation.md:5]: Note that this makes the version number centered, compared to the previous embedded html. I guess that is OK.

**Review by pedberg-icu — APPROVED** [review]

**Comment by pedberg-icu** [comment]
@AEApple @rickmcgowan I just noticed this did not remove "Proposed Update" in the title for tr35.md, I will make another PR to do that.

**Comment by AEApple** [comment]
I can do that, sorry I missed it the first time.

**Comment by pedberg-icu** [comment]
> I can do that, sorry I missed it the first time.

I did that and made a few other fixes: https://github.com/unicode-org/cldr/pull/2465

**Comment by AEApple** [comment]
Thank you!

**Pull request merged**